src/index.html

```diff
@@ -5,8 +5,385 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img
+        src="images/logo.png"
+        alt="moyo-logo"
+        >
+      </a>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--is-active is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smarts-and-phones">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+                  data-qa="nav-hover"
+                  class="nav__link"
+                  href="#laps-and-PC"
```

Comment:
incorrect indentation for transfered line

Reply:
In one of my previous projects mentor showed me this variant of indentation. I don't like it neither but that's how it must be for some reason that I don't know.

Reply:
There should be 2 spaces indent